### PR TITLE
Admin Dashboard UI

### PR DESCRIPTION
**Pull request by abhishek310001**
# Description:

This PR introduces the Admin Dashboard Project of the Odin Project's Intermediate HTML and CSS section. From this project I've implemented the use of grids to customize the layout of the web-page.

The screenshot of UI is attached in this PR for reference

## Implementations:

- Used tailwind CSS to get color actually needed in this project
- Sidebar icon's are SVGs taken from https://pictogrammers.com/library/mdi/
- Profile picture's of users are downloaded from https://www.flaticon.com/free-icons/profile
- Mostly used **Display: grid** property as stated by the project description
- Currently there is dummy content in the **project description** and **announcements** as well

## UI Screenshot:

admin-dashboard

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+                <div class="search">
+                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+                        <path d="M9.5,3A6.5,6.5 0 0,1 16,9.5C16,11.11 15.41,12.59 14.44,13.73L14.71,14H15.5L20.5,19L19,20.5L14,15.5V14.71L13.73,14.44C12.59,15.41 11.11,16 9.5,16A6.5,6.5 0 0,1 3,9.5A6.5,6.5 0 0,1 9.5,3M9.5,5C7,5 5,7 5,9.5C5,12 7,14 9.5,14C12,14 14,12 14,9.5C14,7 12,5 9.5,5Z" />
+                    </svg>
+                    <input type="text">
+                </div>
+                <div class="user-info">
+                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+                        <title>bell-ring-outline</title>
+                        <path d="M10,21H14A2,2 0 0,1 12,23A2,2 0 0,1 10,21M21,19V20H3V19L5,17V11C5,7.9 7.03,5.17 10,4.29C10,4.19 10,4.1 10,4A2,2 0 0,1 12,2A2,2 0 0,1 14,4C14,4.1 14,4.19 14,4.29C16.97,5.17 19,7.9 19,11V17L21,19M17,11A5,5 0 0,0 12,6A5,5 0 0,0 7,11V18H17V11M19.75,3.19L18.33,4.61C20.04,6.3 21,8.6 21,11H23C23,8.07 21.84,5.25 19.75,3.19M1,11H3C3,8.6 3.96,6.3 5.67,4.61L4.25,3.19C2.16,5.25 1,8.07 1,11Z" />
```
Comment (devanuj97) [inline index.html]: this is for ?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+        <div class="header">
+            <div class="search-area">
+                <div class="search">
+                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+                        <path d="M9.5,3A6.5,6.5 0 0,1 16,9.5C16,11.11 15.41,12.59 14.44,13.73L14.71,14H15.5L20.5,19L19,20.5L14,15.5V14.71L13.73,14.44C12.59,15.41 11.11,16 9.5,16A6.5,6.5 0 0,1 3,9.5A6.5,6.5 0 0,1 9.5,3M9.5,5C7,5 5,7 5,9.5C5,12 7,14 9.5,14C12,14 14,12 14,9.5C14,7 12,5 9.5,5Z" />
+                    </svg>
+                    <input type="text">
+                </div>
+                <div class="user-info">
+                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
```
Comment (devanuj97) [inline index.html]: can you find any other way to add css, that way you html will look much lighter

File: index.html (hunk 1 above)
Comment (abhishek310001) [reply index.html]: Its for notification SVG

File: index.html (hunk 2 above)
Comment (abhishek310001) [reply index.html]: Will try to embed using img tag

Comment (devanuj97) [reply index.html]: try implementing here and other place as well

File: style.css [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+    gap: 10px;
+}
+
+.sidebar img{
+    filter: invert(1);
+    height: 25px;
+}
+
+.dashboard-content img{
+    height:50px
```
Comment (devanuj97) [inline style.css]: confirm if we explicitly need to define `width: auto`

File: style.css [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+    align-items: center;
+    justify-self: flex-start;
+}
+
+.btns{
+    display: flex;
+    gap: 20px;
+}
+
+button{
```
Comment (devanuj97) [inline style.css:142]: you can use `shadow` property on button to make it little more clear

File: index.html [hunk 5]
```diff
@@ -0,0 +1,248 @@
...
+            <div class="dashboard-container">
+                <div class="dashboard-content">
+                    <img src="./icons/view-dashboard.svg" alt="Dashboard Icon">
+                    <div class="dashboard-text">
+                        Dashboard
+                    </div>
+                </div>
+            </div>
+            <div class="user-menus">
+                <div class="child">
```
Comment (devanuj97) [inline index.html]: can the naming be little more relevant to the context, as discussed

**Review by devanuj97 — APPROVED** [review]

File: style.css (hunk 3 above)
Comment (abhishek310001) [reply style.css]: the initial value of width is set to `auto` so it doesn't need to be explicitly defined

File: style.css (hunk 4 above)
Comment (abhishek310001) [reply style.css:142]: done

File: index.html (hunk 5 above)
Comment (abhishek310001) [reply index.html]: changed it to relevant name

**Pull request merged**